app/settings/settings.js

```diff
@@ -41,11 +47,30 @@ class Settings {
 
   getCurrentSettings() {
     return {
+      id: this.id,
       onEvent: javaScriptUtils.deepCopyObject(this.onEvent),
       periodic: javaScriptUtils.deepCopyObject(this.periodic),
       isPeriodicMessageSendingActive: this.isPeriodicMessageSendingActive
     };
   }
+
+  loadValuesFromDb() {
+    settingsSchema.find({}, (err, data) => {
+      // `find` returns an array by default.
+      if (javaScriptUtils.isDefined(data[0])) {
```

Comment:
Let's save `data[0]` to a const :) Then probably the comment isn't needed either.